tests/ident/test_engine_parser_msamanda.py

```diff
@@ -116,47 +117,18 @@ def test_engine_parsers_msamanda_unify_row():
                     "name": "Acetyl",
                 },
             ],
-            # "omssa_mod_dir": Path(__file__).parent / "data",
+            "omssa_mod_dir": Path(__file__).parent / "data",
+            "Raw data location": "/Users/cellzome/Dev/Gits/Ursgal/ursgal_master/example_data/test_Creinhardtii_QE_pH11.mzML",
         },
     )
-
-    for row in parser:
-        print(row)
-
-
-# def test_engine_parsers_msamanda_next():
-#     input_file = (
-#         Path(__file__).parent / "data" / "test_Creinhardtii_QE_pH11_omssa_2_1_9.csv"
-#     )
-#     rt_lookup_path = Path(__file__).parent / "data" / "_ursgal_lookup.csv.bz2"
-#     db_path = Path(__file__).parent / "data" / "test_Creinhardtii_target_decoy.fasta"
-
-#     parser = MSamandaParser(
-#         input_file,
-#         params={
-#             "rt_pickle_name": rt_lookup_path,
-#             "database": db_path,
-#             "modifications": [
-#                 "C,fix,any,Carbamidomethyl",
-#                 "M,opt,any,Oxidation",
-#                 "*,opt,Prot-N-term,Acetyl",
-#             ],
-#             "omssa_mod_dir": Path(__file__).parent / "data",
-#         },
-#     )
-#     row = next(parser)
-
-# assert isinstance(row, UnifiedRow)
-# print(row.data.keys())
-# assert row["Sequence"] == "SHCIAEVEK"
-# assert row["Modifications"] == "Carbamidomethyl:3"
-# assert (
-#     row["Raw data location"]
-#     == "/Users/cellzome/Dev/Gits/Ursgal/ursgal_master/example_data/test_Creinhardtii_QE_pH11.mgf"
-# )
-# assert row["Charge"] == "4"
-# assert float(row["Amanda:Score"]) == pytest.approx(83.16219696000763)
-# # assert float(row["uCalc m/z"]) == 0
-# # assert float(row["uCalc mass"]) == 0
-# assert float(row["Calc mass"]) == pytest.approx(3033.491)
-# assert float(row["Calc m/z"]) == pytest.approx(759.38002646677)
+    df = parser.get_dataframe()
+    first_row = df.iloc[0, :]
+    assert first_row["Sequence"] == "SHCIAEVEK"
+    assert first_row["Modifications"] == "Carbamidomethyl:3"
+    assert (
+        first_row["Raw data location"]
+        == "/Users/cellzome/Dev/Gits/Ursgal/ursgal_master/example_data/test_Creinhardtii_QE_pH11.mzML"
+    )
+    assert first_row["Charge"] == 3
+    assert float(first_row["Amanda:Score"]) == pytest.approx(83.16219696000763)
+    assert float(first_row["Exp m/z"]) == pytest.approx(358.174682617188)
```

Comment:
same as above - please assert identity of the resulting df - not just first line